src/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Components using the react-intl module require access to the intl context.
+ * This is not available when mounting single components in Enzyme.
+ * These helper functions aim to address that and wrap a valid,
+ * English-locale intl context around them.
+ */
+import { cloneElement } from 'react';
+import { IntlProvider, intlShape } from 'react-intl';
+import { mount, shallow, render } from 'enzyme';
+
+export const mockIntl = {
+  defaultLocale: 'en',
+  formatDate: () => 'mock date',
+  formatHTMLMessage: ({ id }) => id,
+  formatMessage: ({ id }) => id,
+  formatNumber: value => `${value}`,
+  formatPlural: value => `${value}`,
+  formatTime: value => `${value}`,
+  formatRelative: value => `${value}`,
+  now: () => 0,
+};
+
+const messageProxy = new Proxy({}, { get: (_, property) => property });
+
+// Create the IntlProvider to retrieve context for wrapping around.
+const intlProvider = new IntlProvider({ locale: 'en', messages: messageProxy }, {});
+const { intl } = intlProvider.getChildContext();
+
+const intlNode = node => cloneElement(node, { intl });
```

Comment:
This feels wrong....We are cloning the node we want to unit test? Why can't we pass the i18n context in the shallow/render/mount functions?

Reply:
What happens if a component doesn't take an intl prop like [terra-time-input](https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/src/TimeInput.jsx#L69) which expected intl via context?

Reply:
This is nearly identical to what [react-intl recommends in their docs](https://github.com/yahoo/react-intl/wiki/Testing-with-React-Intl#helper-function-1)

This is just passing the `intl` object as a prop to the subject under test, so components wrapped in `injectIntl` can be tested.

Below, we also pass in the context as the second arg to shallow, mount, etc so that `<FormattedMessage />` etc [work properly](https://github.com/yahoo/react-intl/blob/96a11032768fb3087dcdbe5be0b9e0ba90accb89/src/components/message.js#L79) as well.